docs/sql-reference/sql-statements/data-manipulation/UPDATE.md

```diff
@@ -1,15 +1,36 @@
 # UPDATE
 
-Modifies rows in a Primary Key table. In versions earlier than version 3.0, the UPDATE statement only supports simple syntax, such as `UPDATE <table_name> SET <column_name>=<expression> WHERE <where_condition>`. Starting from version 3.0, StarRocks enriches the syntax to support multi-table joins and common table expressions (CTEs). If you need to join the table to be updated with other tables in the database, you can reference these other tables in the FROM clause or CTE.
+Modifies rows in a Primary Key table.
+
+In versions earlier than version 3.0, the UPDATE statement only supports single-table UPDATE and does not support common table expressions (CTEs). Starting from version 3.0, StarRocks enriches the syntax to support multi-table joins and CTE. If you need to join the table to be updated with other tables in the database, you can reference these other tables in the FROM clause or CTE.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/92ab9c0d-62d9-40c9-8cd7-cda2b1c7e8e5)
should be changed to "CETs", in plural form